extract_unique_sensors_01.py

```diff
@@ -62,19 +62,18 @@ def extract_all_sensors(sensors_data, sensor_name):
     return candidate_lines
 
 # Loop over candidate_lines and extract sensor names
-def splitting_list(sensors_list):
+def splitting_list(sensors_data = read_sensors(file_name, url)):
     """Splits the candidate_lines by dot and append that to sensors_list. It also removes the 
     empty lines.
     """
     sensors_list = []
     candidate_lines = sensors_list
-    sensors_list = extract_all_sensors(sensors_data)
-    unique_sensors = splitting_list(sensors_list)
+    sensors_list = extract_all_sensors(sensors_data, sensor_name=kat_sensor)
```

Comment:
Ok, I am confused you will have to explain to me what you are trying to accomplish with this.